e2e-tests/src/test/helpers.rs

```diff
@@ -0,0 +1,88 @@
+use std::ops::Deref;
+
+use aleph_client::{
+    pallets::balances::BalanceUserApi, AccountId, Connection, KeyPair, Pair, SignedConnection,
+    TxStatus,
+};
+use anyhow::Result;
+use primitives::Balance;
+use rand::Rng;
+
+use crate::config::Config;
+
+/// A wrapper around a KeyPair for purposes of converting to an account id in tests.
+pub struct KeyPairWrapper(KeyPair);
+
+impl Clone for KeyPairWrapper {
+    fn clone(&self) -> Self {
+        Self(KeyPair::new(self.0.signer().clone()))
+    }
+}
+
+impl Deref for KeyPairWrapper {
+    type Target = KeyPair;
+
+    fn deref(&self) -> &Self::Target {
+        &self.0
+    }
+}
+
+impl From<&KeyPairWrapper> for AccountId {
+    fn from(keypair: &KeyPairWrapper) -> Self {
+        keypair.signer().public().into()
+    }
+}
+
+impl From<KeyPairWrapper> for AccountId {
+    fn from(keypair: KeyPairWrapper) -> Self {
+        (&keypair).into()
+    }
+}
+
+/// Creates a copy of the `connection` signed by `signer`
+pub fn sign(conn: &Connection, signer: &KeyPairWrapper) -> SignedConnection {
```

Comment:
Why not make it a method on the instance of `KeyPairWrapper` that you control? 
```rust
impl KeyPairWrapper {
    pub fn signed_connection(&self, conn: &Connection) -> SignedConnection { ... }
}
```

I find 
```rust
key_pair.signed_connection(&conn)
```
to be a better dev experience than
```rust
use helpers::sign;

sign(signer, conn)
```